deploy/iso/minikube-iso/arch/aarch64/package/docker-buildx-aarch64/docker-buildx.mk

```diff
@@ -16,7 +16,10 @@ DOCKER_BUILDX_AARCH64_ENV = \
         GOPATH="$(DOCKER_BUILDX_AARCH64_GOPATH)" \
         GOBIN="$(DOCKER_BUILDX_AARCH64_GOPATH)/bin" \
         PATH=$(DOCKER_BUILDX_AARCH64_GOPATH)/bin:$(BR_PATH) \
-        GOARCH=arm64
+        GOARCH=arm64 \
+	GOPROXY="https://proxy.golang.org,direct" \
+	GOSUMDB='sum.golang.org'\
+	GOOS=linux
```

Comment:
Unify indent?

Reply:
I think the intents here are already the same with other go envs?